### PR TITLE
Added navigation component

### PR DESCRIPTION
**Pull request by jeetsdev**
## Navigation component 

Screenshot from 2022-02-07 23-19-55

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]

File: documentation/docs.html [hunk 1]
```diff
@@ -824,6 +825,57 @@ <h3 class="headline-3">Stacked Item 3</h3>
...
+                <section id="navigations-sec">
+                    <div class="comp-intro">
+                        <h1 class="headline-1 text-head">Navigations</h1>
+                        <p class="headline-3">
+                            Navigation bars are used to show brand logos and links to it's social presences.
+                        </p>
+                    </div>
+                    <div class="comp-details">
+                        <h2 class="headline-2 text-sub-head">Examples of desktop navigatins</h2>
+                        <p class="text-mid">
```
Comment (anjali1102) [inline documentation/docs.html]: Pls fix this typo

File: components/navigations.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    margin: auto;
+}
+
+.nav .nav-search form {
+    border: 1px solid var(--color-grey);
+    gap: 5px;
+}
+
+.nav .nav-search input{
+    margin: 0 .5rem;
```
Comment (anjali1102) [inline components/navigations.css]: you mean 0, 0.5 here?

**Review by anjali1102 — APPROVED** [review]
you can give curser: pointer to fas icons

File: documentation/docs.html (hunk 1 above)
Comment (jeetsdev) [reply documentation/docs.html]: Done, Thanks.

File: components/navigations.css (hunk 2 above)
Comment (jeetsdev) [reply components/navigations.css]: Updated.

**Pull request merged**